common/src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -169,7 +173,9 @@ private void validate() {
                     "`useOptimizations: 'UNWIND_BATCH_PARAMS'` can be used only in combination with `format: 'CYPHER_SHELL' but got [format:`"
                             + this.format + "]");
         }
-        if (!OptimizationType.NONE.equals(this.optimizationType) && this.unwindBatchSize > this.batchSize) {
+        // CSV doesn't use optimization type
+        if (!OptimizationType.NONE.equals(this.optimizationType) && this.unwindBatchSize > this.batchSize
+            && !ExportFormat.CSV.equals(this.format)) {
             throw new RuntimeException("`unwindBatchSize` must be <= `batchSize`, but got [unwindBatchSize:"
                     + unwindBatchSize + ", batchSize:" + batchSize + "]");
         }
```

Comment:
I'd say this message is not good. It's better to split into another condition where we throw a specific exception for the case where we've passed an optimization and a csv format?